### DISCUSSION TITLE
Showing only leaf level in a widget

### DISCUSSION THREAD

**Discussion by ncosgrov58** [atoti/atoti#714] [Q&A] [answered]
I have a hierarchy of Accounts and Projects associated with those Accounts.  When I pull projects over into a pivot row, I will get the account name and the associated projects as leaves (correct functionality) but in some instance on a report I only want the project name without the upper level account Hierarch displayed.  Can I hide the upper level of the hierarchy or just select the leaf level?

**Comment by tibdex** [comment, marked as answer]
Yes, in atoti 0.7.1 you can hide table columns by right-clicking on their header:

![hiding a table column](https://user-images.githubusercontent.com/6574550/197543818-4bf3150f-27d7-4869-8306-fad9501af887.png)
